src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -188,7 +188,6 @@ private CommandResult executeCommand(String commandText) throws CommandException
 
             return commandResult;
         } catch (CommandException | ParseException e) {
-            logger.info("Invalid command: " + commandText);
```

Comment:
What if the command is actually invalid?

Reply:
@damithc Seems like this log statement was already existing before this PR, its possible to split the 2 different exceptions and handle them separately, if the desired behaviour is to differentiate between invalid command and parse exception in the logs